tests/test_3/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+#![no_std]
+use soroban_sdk::{contractimpl, symbol, Bytes, Env};
+
+pub struct Contract;
+
+#[derive(Default)]
+#[repr(packed)]
+pub struct Mem {
+    counter: u32,
+    balance: i128,
+}
+impl Mem {
+    fn to_bytes(&self, env: &Env) -> Bytes {
+        Bytes::from_slice(env, unsafe {
+            core::slice::from_raw_parts(
+                (self as *const Mem) as *const u8,
+                ::core::mem::size_of::<Mem>(),
+            )
+        })
+    }
+    fn from_bytes(b: &Bytes) -> Mem {
+        let mut buf = [0u8; core::mem::size_of::<Mem>()];
+        b.copy_into_slice(&mut buf);
+        unsafe { core::mem::transmute(buf) }
+    }
```

Comment:
@graydon Is it safe to save a packed struct like this and recover it later on? Do you know if the format of what is generated and loaded could change between compiler versions, or depending on host platforms?

Reply:
I'm wondering if for situations where it could make sense for hyper optimization, if we should provide a `#[contracttype(packed = true)]`.

Reply:
it can change between versions and between hosts. this isn't generally safe to be doing.